src/main/java/com/shanebeestudios/skbee/elements/recipe/effects/EffUnregisterRecipe.java

```diff
@@ -0,0 +1,83 @@
+package com.shanebeestudios.skbee.elements.recipe.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import com.shanebeestudios.skbee.api.recipe.RecipeUtil;
+import org.bukkit.Bukkit;
+import org.bukkit.Keyed;
+import org.bukkit.NamespacedKey;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Recipe - Unregister")
+@Description({"Remove a recipe from your server. Recipes can be removed at any time ",
+        "but it is best to do so during a server load event. If a recipe is removed whilst a player is online ",
+        "it will still show up in their recipe book, but they will not be able to craft it. If need be, you can get ",
+        "a list of all recipes by simply typing \"/minecraft:recipe give YourName \" in game.",
+        "You can remove Minecraft recipes, custom recipes and recipes from other plugins. Requires MC 1.13+",
+        "You can remove custom potion recipes by using the potion keyword. Requires PaperMC 1.18+"})
```

Comment:
```suggestion
@Description({"Remove a recipe from your server. Recipes can be removed at any time",
        "but it is best to do so during a server load event. If a recipe is removed whilst a player is online",
        "it will still show up in their recipe book, but they will not be able to craft it. If need be, you can get",
        "a list of all recipes by simply typing \"/minecraft:recipe give YourName \" in game.",
        "You can remove Minecraft recipes, custom recipes and recipes from other plugins.",
        "You can remove custom potion recipes by using the potion keyword. Potions requires PaperMC 1.18+"})
```
Just some formatting stuff.
SkriptHub docs takes care of spaces.